internal/commands/user_run.go

```
@@ -0,0 +1,350 @@
+	fmt.Printf(`User '%s' Info:
+	Display Name:	%s
+	Email:		%s
+	Groups:		%v
+	Disabled:	%v
+`, username, details.GetDisplayName(), strings.Join(details.GetEmails(), ", "), strings.Join(details.GetGroups(), ", "), details.Disabled)
+
+	return nil
+}
+
+// UserAddRunE adds a user.
+//nolint: gocyclo
```

golangci-lint (1.62.2)

File is not `goimports`-ed with -local github.com/authelia/authelia

(goimports)